source/includes/harvest/_jobs.md

```diff
@@ -583,6 +583,66 @@ delete_value  | n/a | When this element is included with a value of "true" (note
 <br>
 [See noteworthy response attributes.](#the-job-object)
 
+## PUT: Replace Hiring Team
+
+```shell
+curl -X PUT 'https://harvest.greenhouse.io/v1/jobs/{id}'
+-H "On-Behalf-Of: {greenhouse user ID}"
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+```json
+{
+    "hiring_managers": [{"user_id": 1234}, {"user_id": 2345}],
+    "sourcers": [{"user_id": 3456}, {"user_id": 4567}],
+    "recruiters": [
+      {
+        "user_id": 5678,
+        "responsible_for_future_candidates:" true,
+        "responsible_for_active_candidates:" true,
+        "responsible_for_inactive_candidates:" true,
+      }, 
+      {
+        "user_id": 6789,
+        "responsible_for_future_candidates:" false,
+        "responsible_for_active_candidates:" false,
+        "responsible_for_inactive_candidates:" false,
+      }
+    ],
+    "coordinators": [
+      {
+        "user_id": 7890,
+        "responsible_for_future_candidates:" false,
+        "responsible_for_active_candidates:" true,
+        "responsible_for_inactive_candidates:" true,
+      }, 
```

Comment:
I think this needs to be 
+        `"responsible_for_future_candidates:" **true**,
 +        "responsible_for_active_candidates:" true,
 +        "responsible_for_inactive_candidates:" true,`

There isn't a way to make do this in the UI right now. When making someone responsible (for either role), these are the only options:

1. False/False/False
2. True/False/False
3. True/True/False
4. True/False/True
5. True/True/True